FusionIIIT/applications/academic_procedures/views.py

```diff
@@ -2271,6 +2213,18 @@ def student_list(request):
         obj = json.dumps(maindict)
         return HttpResponse(obj, content_type='application/json')
 
+@login_required(login_url='/accounts/login')
+def course_list(request):
```

Comment:
Merging it now, but fix this later.
There is no need of this view, pass the data via student homepage view only